shoplist/src/main/java/com/netguru/codereview/network/ShopListRepository.kt

```diff
@@ -0,0 +1,10 @@
+package com.netguru.codereview.network
```

Comment:
Repository should not be in network package. It should not know where it is getting data from (DB or API)